views/products/presentation/tandem_monero.tpl

```diff
@@ -0,0 +1,49 @@
+<h4>Introduction</h4>
+
+<p>
+    With the upcoming Web 3.0, more and more services rely on private keys.
+    This includes, but is not limited to, blockchains and identity management systems
+    that allow a user to selectively disclose one or more attributes.
+    Another element of Web 3.0 is decentralization, which makes it harder
+    to revoke a stolen or lost private key.
+</p>
+
+<p>
+    The most common protections for these private keys include
+    <i>Wallets as a service</i> like hardware wallets and hardware
+    security modules.
+    But these solutions all contain a single point of failure that can
+    leak the private key or lead to abuse.
+    To avoid a single point of failure, one can use threshold
+    cryptographic solutions like Calypso or Sepior [https://sepior.com/].
```

Comment:
You mean in SPRING/projects.yaml ? I prefer having only links in there that are actual part of the project.

Reply:
> You mean in SPRING/projects.yaml ? I prefer having only links in there that are actual part of the project.

I wasn't clear (such a small commnent), I meant: change "Sepior" to a link instead of an inline url